src/main/java/reposense/git/GitLog.java

```diff
@@ -10,10 +10,12 @@
 
 /**
  * Contains git log related functionalities.
- * Git log is responsible to obtain the commit logs and the authors' info.
  */
 public class GitLog {
 
+    /**
+     * Responsible for obtaining the commit logs and the authors' info.
+     */
     public static String get(RepoConfiguration config, Author author) {
```

Comment:
The Javadocs for methods, should be more of explaining what the method is doing, rather than just saying what it is responsible for. Like in this case, it should be saying something like:
`Returns the git commit log info of {@code Author}, in the repository specified in {@code config}`